core/common/src/main/java/alluxio/RuntimeConstants.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
what was the circular dependency and how does it manifest in Jenkins?

would making Runtime a nested class of Constants work?